modules/appeals_api/app/controllers/appeals_api/contestable_issues/v0/contestable_issues_controller.rb

```diff
@@ -26,20 +29,5 @@ def schema
     def token_validation_api_key
       Settings.dig(:modules_appeals_api, :token_validation, :contestable_issues, :api_key)
     end
-
-    def form_schema
-      @form_schema ||= AppealsApi::FormSchemas.new(
-        SCHEMA_ERROR_TYPE,
-        schema_version: 'v2'
-      )
-    end
-
-    def request_headers
-      self.class::HEADERS.index_with { |key| request.headers[key] }.compact
-    end
-
-    def validate_json_schema_for_headers
-      form_schema.validate!(self.class::FORM_NUMBER, request_headers)
-    end
```

Comment:
I ended up refactoring this after finding a simpler way (`self.class::CONSTANT`) that avoided overriding so many parent methods.

Something that still seems "off" is that `FORM_NUMBER` isn't a very relevant constant name for Contestable Issues (or Legacy Appeals). I think we're using that because HLR/NOD/SC use that convention. Those endpoints also have `FORM_NUMBER` equal to the form name/number without "_HEADERS" appended at the end. I also wonder if the `schema` endpoint is working correctly because I think it's loading the header schema rather than the form schema. I wasn't sure of the intent, so I didn't change it here. Possibly some work for a future PR.